test/fork.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {describe, it} from 'mocha';
+import {logger, octokit} from './util';
+import * as sinon from 'sinon';
+import * as handler from '../src/github-handler';
+import * as responseData from './fixtures/create-fork-response.json';
```

Comment:
nit: let's move this closer to the test (in the `it` or `describe` block) rather than as a global variable.

```
const responseData = require('path/to/file');
```